packages/ui/layout/BottomNavBar/index.tsx

```diff
@@ -1,17 +1,62 @@
 import { Flex } from '@chakra-ui/react';
 
+import AnimalsSelectedIcon from '../../assets/bottomNavBar/icon_animals_selected.svg';
+import AnimalsUnselectedIcon from '../../assets/bottomNavBar/icon_animals_unselected.svg';
+import ChattingsSelectedIcon from '../../assets/bottomNavBar/icon_chattings_selected.svg';
+import ChattingsUnselectedIcon from '../../assets/bottomNavBar/icon_chattings_unselected.svg';
+import MyPageSeletedIcon from '../../assets/bottomNavBar/icon_mypage_selected.svg';
+import MyPageUnselectedIcon from '../../assets/bottomNavBar/icon_mypage_unselected.svg';
+import VolunteersSelectedIcon from '../../assets/bottomNavBar/icon_volunteers_selected.svg';
+import VolunteersUnselectedIcon from '../../assets/bottomNavBar/icon_volunteers_unselected.svg';
+import NavBarButton from './NavBarButton';
+import { useBottomNavBar } from './useBottomNavBar';
+
 export default function BottomNavBar() {
+  const { selected, goVounteers, goAnimals, goChattings, goMyPage } =
+    useBottomNavBar();
+
   return (
-    <Flex
-      w="100%"
-      h="50px"
-      justify="center"
-      align="center"
-      bottom={0}
-      pos="fixed"
-      zIndex={10}
-    >
-      BottomNavBar
-    </Flex>
+    selected && (
+      <Flex
+        bgColor="white"
+        w="100%"
+        h="50px"
+        px={4}
+        justify="center"
+        justifyContent="space-between"
```

Comment:
```suggestion
        justify="space-between"
```

`justify`와 `justifyContent` 모두 같은 의미를 지니는 속성입니다!! `justify="space-between"`으로 수정되면 좋을 것 같아요!!

Reply:
오, 이부분은 제가 실수 한 것 같습니다!
바로 수정하도록 하겠습니다~